components/homme/src/preqx/derivative_mod.F90

```diff
@@ -6,7 +6,7 @@ module derivative_mod
   use derivative_mod_base, only: derivative_t, subcell_integration, subcell_dss_fluxes, subcell_div_fluxes,                  &
        subcell_Laplace_fluxes, allocate_subcell_integration_matrix, derivinit, gradient, gradient_wk, vorticity, divergence, &
        gradient_sphere_wk_testcov, gradient_sphere_wk_testcontra, ugradv_sphere, vorticity_sphere, vorticity_sphere_diag,    &
-       curl_sphere,curl_sphere_wk_testcov, vlaplace_sphere_wk, element_boundary_integral, edge_flux_u_cg,                    &
```

Comment:
Just curious, what was edge_flux_u_cg for?

Reply:
i think it was used in the DG code